### PR TITLE
Add aria labels and consolidate loc strings

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Added aria labels for the tables and columns that didn't have display text.

Also consolidated loc strings into single file for ease of re-use and ensuring no duplicates.

### CODE REVIEW

File: extensions/big-data-cluster/src/bigDataCluster/localizedConstants.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
+/*---------------------------------------------------------------------------------------------
```
Comment (alanrenmsft) [inline extensions/big-data-cluster/src/bigDataCluster/localizedConstants.ts]: I remembered I did this once and @kevcunnane left a comment saying we should avoid this. @kevcunnane do you recall the reason?

**Comment by Benjin** [comment]
I personally try to split up changes that aren't inextricably linked into different PRs so that people can approve or reject things piecemeal (Alan's comment), and to make it easier for reviewer to identify which changes are for what end-goal.  I may have scrolled through a little too quickly, but I missed where the aria label additions were amid the loc string consolidations.

**Comment by Charles-Gagnon** [comment]
@Benjin - Yeah, sorry about that. I definitely agree - this was a case of starting to go down the rabbit hole of moving the strings part of the way through and since the ariaLabel was going to use that I didn't want to deal with the merge conflicts afterwards. 

Here's where the labels were added

https://github.com/microsoft/azuredatastudio/pull/8494/files#diff-9ca0f860ba9eaa43586fbe3620c78991R126
https://github.com/microsoft/azuredatastudio/pull/8494/files#diff-9ca0f860ba9eaa43586fbe3620c78991R195
https://github.com/microsoft/azuredatastudio/pull/8494/files#diff-9ca0f860ba9eaa43586fbe3620c78991R212
https://github.com/microsoft/azuredatastudio/pull/8494/files#diff-9ca0f860ba9eaa43586fbe3620c78991R290
https://github.com/microsoft/azuredatastudio/pull/8494/files#diff-9ca0f860ba9eaa43586fbe3620c78991R307
https://github.com/microsoft/azuredatastudio/pull/8494/files#diff-984cd235f47ce2a66ebf10de6b157626R80
https://github.com/microsoft/azuredatastudio/pull/8494/files#diff-984cd235f47ce2a66ebf10de6b157626R149
https://github.com/microsoft/azuredatastudio/pull/8494/files#diff-984cd235f47ce2a66ebf10de6b157626R166
https://github.com/microsoft/azuredatastudio/pull/8494/files#diff-984cd235f47ce2a66ebf10de6b157626R266

**Comment by Benjin** [comment]
Awesome; thanks for the links, @Charles-Gagnon

**Pull request merged**